### PR TITLE
Updates and fixes throughout the document

### PR DESCRIPTION
**Pull request by jorainer**

### CODE REVIEW

File: vignettes/xcms-preprocessing.Rmd [hunk 1]
```diff
@@ -1272,16 +1272,17 @@ similar retention time will result in a higher peak density at a certain
 retention time and are thus grouped together. The grouping depends on the
 *smoothness* of the density curve and can be configured with parameter `bw`.
 
-An illustration of this is shown in the sketch below.
+An illustration how chromatographic peaks within a small *m/z* range are grouped
```
Comment (philouail) [inline vignettes/xcms-preprocessing.Rmd:1275]: "An Illustration SHOWING how..." or something similar ?

**Review by philouail — CHANGES_REQUESTED** [review]
just fix that one comment

File: vignettes/xcms-preprocessing.Rmd (hunk 1 above)
Comment (jorainer) [reply vignettes/xcms-preprocessing.Rmd]: Fixed now.

**Comment by jorainer** [comment]
Thanks for looking through the changes @philouail ! I made the requested change and re-requested your review.

**Review by philouail — APPROVED** [review]
All good

**Pull request merged**